### DISCUSSION TITLE
STUMPY for PULSAR time series.

### DISCUSSION THREAD

**Discussion by indrajitbarve** [TDAmeritrade/stumpy#348] [General]
Hello, I am impressed with your package. and I would like to discuss with you regarding using STUMPY for the PULSAR time series. 

Thanks and regards 
Indrajit

**Comment by seanlaw** [comment]
@indrajitbarve Welcome to the STUMPY community. I'm not familiar with the "PULSAR time series" so can you please provide some context and details so that I or others may be able to chime in?

**Reply by indrajitbarve** [reply]
Hello,


A pulsar is a pulsating star. The pulse signal from the star goes through dispersion, scattering, etc.  Building back the pulse signal is what I am looking for. While doing this Radio Frequency Interference (RFI) which is a random locally generated signal spoiling the detection. I need to classify the RFI and mask it before processing. 

please go through this. http://www.ncra.tifr.res.in/~ruta/ras15/yogesh_ras2015.pdf.
